### ISSUE TITLE
manager failed to create user

### ISSUE THREAD

**Issue by FengZh61** [nilsteampassnet/TeamPass#4540]
### Steps to reproduce
1. Create a user with manager privileges
2. I used it to create a user, logged in and created the first user successfully, then continued to create, clicked submit and went back to the user page, the user had not been created.
3. If I log out and log back in, it's still the same as step 2
4. If you change the user to teampass manager first and then to manager, manger is able to create USER users consecutively.
### Expected behaviour
```
// if ((int) $session->get('user-manager') === 1
        //     && !in_array($targetUserInfos['isAdministratedByRole'], $session->get('user-roles_array'))) {

        //     echo prepareExchangedData(
        //         array(
        //             'error' => true,
        //             'message' => $lang->get('error_not_allowed_to'),
        //         ),
        //         'encode'
        //     );
        //     exit;
        // }
```
By commenting out this code, the successive creation of the user user is successful

### Actual behaviour
Tell us what happens instead

### Server configuration
**Operating system**: CentOS Linux 7 (Core)

**Web server:** Apache/2.4.6 (CentOS)

**Database:** 11.5.1-MariaDB

**PHP version:** 8.1.29

**Teampass version:** 3.1.3.1

**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
[Mon Dec 23 14:52:31.092435 2024] [php:warn] [pid 72064] [client 192.168.159.1:61452] PHP Warning:  Trying to access array offset on value of type null in /var/www/html/teampass/sources/users.queries.php on line 153, referer: https://192.168.159.161/teampass/index.php?page=users
[Mon Dec 23 14:52:31.092473 2024] [php:warn] [pid 72064] [client 192.168.159.1:61452] PHP Warning:  Trying to access array offset on value of type null in /var/www/html/teampass/sources/users.queries.php on line 154, referer: https://192.168.159.161/teampass/index.php?page=users
[Mon Dec 23 14:52:31.092477 2024] [php:warn] [pid 72064] [client 192.168.159.1:61452] PHP Warning:  Trying to access array offset on value of type null in /var/www/html/teampass/sources/users.queries.php on line 155, referer: https://192.168.159.161/teampass/index.php?page=users
[Mon Dec 23 14:52:31.092481 2024] [php:warn] [pid 72064] [client 192.168.159.1:61452] PHP Warning:  Trying to access array offset on value of type null in /var/www/html/teampass/sources/users.queries.php on line 169, referer: https://192.168.159.161/teampass/index.php?page=users
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by DobroFenix** [comment]
Confirm. The manager works very crookedly.

**Comment by corentin-soriano** [comment]
Be aware that commenting out this block of code introduces a vulnerability that allows a manager to manage accounts that they should not.
I will try to improve this.

**Comment by corentin-soriano** [comment]
Can you try this fix please? #4541

**Comment by FengZh61** [comment]
> Be aware that commenting out this block of code introduces a vulnerability that allows a manager to manage accounts that they should not. I will try to improve this.

This is a temporary solution

**Comment by FengZh61** [comment]
> Can you try this fix please? #4541

Okay, I'll test it and get back to you.

**Comment by FengZh61** [comment]
> Can you try this fix please? #4541

Runs fine, successive new USER users, works fine, delete and edit functions are also fine, thanks for the fix, very great update!
